### ISSUE TITLE
Archi 4.9.0 EA4 terminates while exporting model to HTML

### ISSUE THREAD

**Issue by startsevsa** [archimatetool/archi#760]
### Version of Archi
4.9.0 EA 4

### Archi Plug-ins
coArchi
jArchi

### Operating System
Windows 10

### Expected Behaviour
Successful export finishing, closing modal progress window and returning to the main Archi window.

### Actual Behaviour
When exporting model to HTML (showing a modal window with a progress bar for some seconds) Archi quits without any message.
[hs_err_pid14304.log](https://github.com/archimatetool/archi/files/7099849/hs_err_pid14304.log)
[hs_err_pid11876.log](https://github.com/archimatetool/archi/files/7099850/hs_err_pid11876.log)

### Steps to Reproduce the Behaviour
1. Export model to HTML

**Comment by Phillipus** [comment]
Thanks for reporting this.

Yep. A mixture of gradient and fill opacity less than 255 does it, even without exporting to HTML report.

I guess this must be (another) Eclipse bug that I'll have to report...

**Comment by Phillipus** [comment]
Please vote for this Eclipse bug!

https://bugs.eclipse.org/bugs/show_bug.cgi?id=575778

**Comment by Phillipus** [comment]
@startsevsa Once again thanks for reporting this bug. Despite extensive testing I missed this one, and it only occurs on Windows.

This is a bug caused by a "fix" [sic] in Eclipse - https://bugs.eclipse.org/bugs/show_bug.cgi?id=121220

I've reported the issue here - https://bugs.eclipse.org/bugs/show_bug.cgi?id=575778

I've also managed to apply a workaround - https://github.com/archimatetool/archi/commit/efdd12cd8b96a7f44cb0bc9a0d7bffa9ca7db461 - this will be in in Archi 4.9 EA 5. In the meantime, don't use a gradient with fill opacity less than 255.

Sadly, these regressions are becoming very common in Eclipse. Someone ought to introduce their developers to the concept of...testing.
